system/jlib/jptree.cpp

```diff
@@ -1057,7 +1057,7 @@ void PTree::setName(const char *_name)
         name = NULL;
     else
     {
-        if (!validateXMLTag(_name)) 
```

Comment:
This unrelated change snuck in, and can't revert since the extra space doesn't pass our check-in criteria